connectors/mcp_base.py

```
@@ -18,367 +17,370 @@
-            'action': 'context_sync',
-            'timestamp': datetime.utcnow().isoformat(),
-            'source': self.connector_id
-        }]
-        
+        merged.history = remote_context.history + [
+            {
+                "action": "context_sync",
+                "timestamp": datetime.utcnow().isoformat(),
+                "source": self.connector_id,
+            }
+        ]
```

Ruff (0.11.9)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)